blocks/v2-footer/v2-footer.css

```diff
@@ -0,0 +1,245 @@
+  font-size: 16px;
+  letter-spacing: 0.5px;
+}
+
+.v2-footer ul {
+  margin: 0;
+  padding: 0;
+  list-style: none;
+}
+
+.v2-footer-links-wrapper :is(h3, li, p, a) {
+  margin-bottom: 0;
+  color: var(--c-main-black);
+}
+
+.v2-footer-list h3 {
+  font-family: var(--ff-volvo-novum-medium);
+  border-top: 1px solid var(--border-color);
+  cursor: pointer;
+  padding: 28px 16px;
+  display: flex;
+  align-items: center;
+}
+
+.v2-footer-list-item {
+  transition: 300ms max-height ease;
+  will-change: max-height;
+  overflow: hidden;
+  max-height: 0;
+}
+
+.v2-footer-list-item a {
+  display: flex;
+  padding-left: 16px;
+  padding-right: 16px;
+  padding-bottom: 16px;
+}
+
+.icon-arrow-down {
+  margin-left: auto;
+  transition: transform var(--duration-medium) cubic-bezier(0.45, 0, 0.125, 1);
+}
+
+.v2-footer-list.expand .icon-arrow-down {
+  transform: rotate(-180deg);
+}
+
+.icon-newtab,
+.icon-fb,
+.icon-twitter,
+.icon-yt,
+.icon-instagram,
+.icon-linkedin {
+  display: none;
+}
+
+.v2-footer-copyright-wrapper {
+  border-top: 1px solid var(--border-color);
+}
+
+.v2-footer-copyright-container {
+  display: flex;
+  padding: 12px 16px;
+  flex-flow: column;
+  align-items: center;
+  gap: 6px;
+  flex-wrap: wrap;
+}
+
+.v2-footer-copyright {
+  text-align: center;
+  flex-basis: 100%;
+}
+
+.v2-footer-bar {
+  display: flex;
+  padding-top: 12px;
+  padding-bottom: 12px;
+  justify-content: flex-end;
+  align-items: center;
+  flex-wrap: wrap;
+}
+
+.v2-footer-bar li {
+  flex: 1 0 auto;
+}
+
+.v2-footer-bar a,
+.v2-footer-copyright p {
+  color: var(--volvo-text-gray);
+  margin: 0;
+  padding-top: 8px;
+  padding-bottom: 8px;
+}
+
+.v2-footer a {
+  text-decoration: none;
+}
+
+.v2-footer a:hover {
+  color: var(--c-cta-blue-hover);
+  text-decoration: underline;
+}
+
+.v2-footer-bar li:not(:first-child) {
+  padding-left: 8px;
+}
+
+.v2-footer-bar li:not(:last-child) {
+  border-right: 1px solid var(--volvo-text-gray);
+  padding-right: 8px;
+}
+
+.v2-scroll-to-top {
+  background-color: var(--c-white);
+  border-radius: 50%;
+  border: 1px solid var(--c-grey-400);
+  bottom: 20px;
+  box-shadow: 0 4px 70px 0 rgb(0 0 0 / 20%);
+  color: var(--c-main-black);
+  cursor: pointer;
+  display: none;
+  padding: 10px;
+  position: fixed;
+  right: 30px;
+  z-index: 99;
+}
+
+.v2-scroll-to-top:hover {
+  background-color: var(--c-grey-50);
+}
+
+.v2-scroll-to-top:active {
+  background-color: var(--c-grey-100);
+}
+
+.v2-footer-list:not(.expand) h3:focus,
+.v2-footer-list-item a:focus,
+.v2-footer-bar a:focus,
+.v2-scroll-to-top:focus {
+  outline: 2px solid var(--light-border-focus);
+}
+
+.v2-scroll-to-top svg {
+  display: block;
+}
+
+/* desktop */
+@media screen and (min-width: 1200px) {
+  .v2-footer-links-wrapper,
+  .v2-footer-copyright-container {
+    width: calc(var(--wrapper-width) - 100px);
+    margin: 0 auto;
+  }
+
+  .v2-footer-links-wrapper {
+    padding-top: 55px;
+    padding-bottom: 55px;
+  }
+
+  .v2-footer-list h3 {
```

Comment:
I would recommend styling based in classes, not in tags, because if the tag is changed by the editor, the style won't be applied.

Reply:
Done